dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/audit/AbstractEventAuditPostProcessor.java

```diff
@@ -0,0 +1,41 @@
+package org.hisp.dhis.dxf2.events.importer.audit;
+
+import java.time.LocalDateTime;
+
+import org.hisp.dhis.artemis.audit.Audit;
+import org.hisp.dhis.artemis.audit.AuditManager;
+import org.hisp.dhis.artemis.audit.AuditableEntity;
+import org.hisp.dhis.audit.AuditScope;
+import org.hisp.dhis.audit.AuditType;
+import org.hisp.dhis.dxf2.events.event.Event;
+import org.hisp.dhis.dxf2.events.importer.EventImporterUserService;
+import org.hisp.dhis.dxf2.events.importer.Processor;
+import org.hisp.dhis.dxf2.events.importer.context.WorkContext;
+import org.hisp.dhis.dxf2.events.importer.mapper.ProgramStageInstanceMapper;
+import org.hisp.dhis.program.ProgramStageInstance;
+
+public abstract class AbstractEventAuditPostProcessor implements Processor
```

Comment:
minor: can we add a short Javadoc to explain the reason for this Post processor

Reply:
Sure, I'll add it, thank you